trade-finance-manager-ui/templates/team-checking/team-checking-page.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "index.njk" %}
```

Comment:
Is there a design for this please, can you point me to the Miro design.

Reply:
I wonder if this is needed, will the user not be see their role type in profile.
Not a big fan of creating temporary pages not meant for production usage.

Reply:
I think lets leave it for now and post first round of QA we can remove it.